### DISCUSSION TITLE
Connect to motioneye

### DISCUSSION THREAD

**Discussion by postelrich** [fuatakgun/eufy_security#237] [Q&A] [answered]
Thanks for this plugin, got it connected to my cameras. How can I add the cameras to motioneye? Is there a fixed url from the rtsp addon for each camera? I have the doorbell that continuously records but only goes back a few days.

**Comment by fuatakgun** [comment, marked as answer]
Yes, for each camera there is a sensor called as streaming address, if you are always using same streaming technology (rtsp vs p2p), URLs are fixed.
